embulk-ruby/lib/embulk/java_plugin.rb

```diff
@@ -1,5 +1,6 @@
 module Embulk
 
+  require 'jruby'
```

Comment:
Questions:
* How is `require 'jruby'` related to `require 'java'`? Doesn't this conflict with `'java'`?
* Doesn't older JRuby have no problems even with `require 'jruby'`?
* Is it the right place to add this `require 'jruby'`? Why not the same place with `require 'java'`?

Reply:
@dmikurube Thank you for your comment.

> Doesn't older JRuby have no problems even with `require 'jruby'`?

I tested this PR using JRuby 9.1.15.0, 9.2.20.1, and 9.3.4.0 (Please see the log above)
The rest of the two comments will answer later.

Reply:
They look to be tested based on Java plugins (even though they are packaged in Gems). JRuby is used only to load Gems there.

Are they really okay with 100% Ruby plugins?

Reply:
Especially in maintaining JRuby-related things in Embulk, I'm negative to make changes by "Tested, worked, then let's go with it."  We've often faced unexpected breakage by such changes.

We wanted to understand :
* What is the background in JRuby.
* How that part is implemented inside JRuby version-to-version.